CHANGELOG.rst

```diff
@@ -1,6 +1,18 @@
 Changelog
 =========
 
+1.11.1 (2022-10-28)
+-------------------
+
+Bug Fixes
+"""""""""
+
+- Fix AWS session token to be considered while AWS secret and access key passed via the connection UI field
+  (`#730 <https://github.com/astronomer/astronomer-providers/pull/730>`_)
+- Fix the return response key of SageMaker Training and Transform Operator
+  (`#737 <https://github.com/astronomer/astronomer-providers/pull/737>`_)
```

Comment:
These descriptions need to be more user-centric

Reply:
Fixed